tests/auto/start_ci_py_pro.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash --login
```

Comment:
I worry a little about having a login shell here. The role accounts often have some shifty stuff in their .rc files. 😬 

Since we are currently including a machine case statements here (we may want to revisit that in the future), would you be up for sourcing the path to the lmod file in this script so we don't inherit all the login "features" that could be difficult to debug?

Actually, I am seeing that the regional_workflow has found a more generic way of doing this as a recent PR that boils down to something like:

```
if [ -z ${ENV_INIT_SCRIPTS_FPS:-""} ]; then
  ENV_INIT_SCRIPTS_FPS=( "/etc/profile" )
fi
for fp in ${ENV_INIT_SCRIPTS_FPS[@]}; do
  source $fp
done
```

Reply:
I agree in principle. I know that this concern has been raised before. I haven't seen this more generic way of doing this, and I'll explore that and make sure it works and I understand it.

Reply:
I agree in principle. I know that this concern has been raised before. I haven't seen this more generic way of doing this, and I want to explore it to make sure it works and I understand it.

Reply:
I believe the functionality we want is to duplicate the sequence of init scripts that gets sourced for a login shell, except we want to skip the ones owned by the user.  I am pretty sure (see: https://askubuntu.com/questions/463462/sequence-of-scripts-sourced-upon-login) that all we want to do is source `/etc/profile`.